app/src/main/java/org/wikipedia/analytics/eventplatform/MachineGeneratedArticleDescriptionsAnalyticsHelper.kt

```diff
@@ -0,0 +1,84 @@
+package org.wikipedia.analytics.eventplatform
+
+import android.content.Context
+
+object MachineGeneratedArticleDescriptionsAnalyticsHelper {
+
+    fun articleDescriptionEditingStart(context: Context) {
+        EventPlatformClient.submit(
+            BreadCrumbLogEvent(
+                BreadCrumbViewUtil.getReadableScreenName(context),
+                "ArticleDescriptionEditing.start"
+            )
+        )
+    }
+
+    fun articleDescriptionEditingEnd(context: Context) {
+        EventPlatformClient.submit(
+            BreadCrumbLogEvent(
+                BreadCrumbViewUtil.getReadableScreenName(context),
+                "ArticleDescriptionEditing.end"
+            )
+        )
+    }
+
+    fun suggestedDescriptionsButtonShown(context: Context) {
+        EventPlatformClient.submit(
+            BreadCrumbLogEvent(
+                BreadCrumbViewUtil.getReadableScreenName(context),
+                "MachineGeneratedArticleSuggestion.suggestedDescriptionsButton.shown"
```

Comment:
Not sure but feels like we can at least make the `MachineGeneratedArticleSuggestion` and `MachineGeneratedArticleSuggestionsApiResponseDetails` as `const` values so we can reduce possible issues in the future.

Reply:
@cooltey this will be the value sent to the mep schema... and will not be reused anywhere. So is it ok if we just leave it this way?

Reply:
Since the same string is used many times in this file, I think it should be better if we can set the string as `const` values so that you have a better way to control it in the future.

Reply:
Done @cooltey